src/System.CommandLine/Builder/CommandLineBuilderExtensions.cs

```diff
@@ -9,59 +9,48 @@
 using System.CommandLine.Parsing;
 using System.IO;
 using System.Threading;
-using System.Threading.Tasks;
-using static System.Environment;
 using Process = System.CommandLine.Invocation.Process;
 
 namespace System.CommandLine
 {
-    /// <summary>
-    /// Provides extension methods for <see cref="CommandLineBuilder"/>.
-    /// </summary>
-    public static class CommandLineBuilderExtensions
+    public partial class CommandLineBuilder
```

Comment:
to keep the diff smaller, I've decided to make the class partial rather than moving all the code to `CommandLineBuilder`